docs/get-started.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Get Started
```

Comment:
"Get started"

Google Developer Style Guide says to use sentence-case for headings.